src/test/java/edu/hm/hafner/analysis/ListTest.java

```diff
@@ -0,0 +1,91 @@
+package edu.hm.hafner.analysis;
+
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests the class {@link ArrayList}.
+ *
+ * @author Elena Lilova
+ */
+
+public abstract class ListTest {
+
+    /**
+     * Creates a new instance of a {@link List}
+     *
+     * @param numberOfInitialElements the number of initial elements
+     * @return a new instance of a {@link List}
+     */
+
+    abstract List<Integer> create(int numberOfInitialElements);
+
+    @Test
+    public void testIsItEmpty() {
+        List list = create(0);
+        assertThat(list.isEmpty());
```

Comment:
```suggestion
        assertThat(list).isEmpty();
```
Sonst fällt die Prüfung weg